pyproject.toml

```diff
@@ -136,6 +138,9 @@ dev-dependencies = [
     "mypy>=1.10.0",
     "celery>=5.4.0",
     "testcontainers",
+    "mysql-connector-python~=8.0",
+    "pymysql",
+    "cryptography",
```

Comment:
I see `mysql-connector-python` being used, but what about these 2?

Reply:
These dependencies are required for using MySqlContainer in Python 3.8